packages/zowe-explorer/src/Profiles.ts

```diff
@@ -283,7 +283,7 @@ export class Profiles extends ProfilesCache {
         const quickpick = vscode.window.createQuickPick();
         const placeholder = localize(
             "addSession.quickPickOption",
-            'Choose "Create new..." to define a new profile or select an existing profile to Add to the USS Explorer'
+            'Choose "Create new..." to define a new profile or select an existing profile to add to the DATA SETS Explorer'
```

Comment:
Could we rephrase it to 'Create new..." to define a new profile or select an existing profile to Add to the Explorer'. We could leave out what type of profile we want. 

Or do you want it to be 'Create new..." to define a new profile or select an existing profile to Add to the DATA SETS, USS or Job Views Explorer'?

Reply:
I know it is sort of a big change, but could you create a text that changes based on which tree the user clicked on? I think you can use the `zoweFileProvider` to decide which tree was clicked. Imho it's important to tell the user that the profile will be added only to the one tree, and not all 3.

Probably we should shorten the whole message as well, since it doesn't fit in the box. If this suggested change is too big, I'm also ok with the shorter, generic message.

Reply:
I changed the display message based on the tree that was clicked using zoweFileProvider. I also shortened the message to fit the display box.